### ISSUE TITLE
Alignment of button on large screens

### ISSUE THREAD

**Issue by cstiens** [digiresilience/landing#3]
![lg-screen-alignment](https://user-images.githubusercontent.com/1434597/39935097-8cf2bf56-550d-11e8-8361-110200e02da6.png)

**Comment by tinabeans** [comment]
@cstiens @kckthx 

OK so I looked into this! And the answer is a little complicated, and it has to do with how things are positioned on the web, as opposed to in a graphics tool like Sketch.

Text on the web lives in boxes, and within those boxes the browser is in charge of determining where a linebreak occurs based on what fits on each line within those boxes. Here is a screenshot that highlights where the box that holds "Your link to a stronger digital community" begins and ends:

<img width="1429" alt="screen shot 2018-05-11 at 2 45 57 pm" src="https://user-images.githubusercontent.com/793711/39941561-f471727c-552a-11e8-82df-d0669b9e32bd.png">

Now the way the "Meet Our Team" button beneath it is aligned is based on the width of its own containing box, which is the same width as the box above it:

<img width="1414" alt="screen shot 2018-05-11 at 2 55 37 pm" src="https://user-images.githubusercontent.com/793711/39941809-b8ad37f2-552b-11e8-8eac-7c93536e4378.png">

Unfortunately, there is no way for the "Meet Our Team" button to know the exact positioning of the word "stronger" and be right aligned to that. It can only be right-aligned within its own container.

That said, would the following be a good graphical solution? I've extended the white bar to 50% of the screen width, to match the width of the containers holding the text. It creates a nice vertical rhythm and alignment that was previously lacking.

<img width="1429" alt="screen shot 2018-05-11 at 2 48 55 pm" src="https://user-images.githubusercontent.com/793711/39941836-ccd3ecb2-552b-11e8-9c7b-a1334561914f.png">


Let me know if this fix works!
